### ISSUE TITLE
Request to create new repository kubernetes-sigs/windows-gmsa

### ISSUE THREAD

**Issue by ddebroy** [kubernetes/org#451]
### New Repo, Staging Repo, or migrate existing
new repository

### Requested name for new repository
windows-gmsa

### Which Organization should it reside
kubernetes-sigs

### If not a staging repo, who should have admin access
@ddebroy @michmike @PatrickLang 

### If not a staging repo, who should have write access
@ddebroy @michmike @PatrickLang 

### If a new repo, who should be listed as approvers in OWNERS
@ddebroy @michmike @PatrickLang 

### If a new repo, who should be listed in SECURITY_CONTACTS
@ddebroy @michmike @PatrickLang 

### What should the repo description be
External components to support Windows GMSA in Kubernetes

### What SIG and subproject does this fall under in sigs.yaml
This is a new subproject for sig-windows called windows-gmsa

### Approvals
Discussed the need for this repo to host GMSA related CRD and Webhooks as outlined in KEP [https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20181221-windows-group-managed-service-accounts-for-container-identity.md] with SIG Windows owners @PatrickLang and @michmike in SIG Windows meeting on Tuesday Feb 5th.

### Additional context for request
This repo will need to host and build the CRD YAMLs and Webhooks associated with GMSA support in Kubernetes as detailed in KEP [https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20181221-windows-group-managed-service-accounts-for-container-identity.md]

**Comment by michmike** [comment]
approved from my end

**Comment by ddebroy** [comment]
Hi @cblecker can you PTAL at this request? Thanks!

**Comment by spiffxp** [comment]
https://github.com/kubernetes/org/pull/469 needs to merge and take effect before I can update the repo's ACLs

https://github.com/kubernetes/community/pull/3246 should be an easy lgtm

**Comment by michmike** [comment]
created a new PR https://github.com/kubernetes/community/pull/3257 to finalize this work. PTAL

**Comment by michmike** [comment]
https://github.com/kubernetes/community/pull/3262 is now merged. so is https://github.com/kubernetes/org/pull/469

@spiffxp  can you please move this forward? thanks!

**Comment by spiffxp** [comment]
/close
Added ACLs, and thanks for fixing my typo in the sigs.yaml PR.

FWIW, I hope you weren't waiting on these to move forward.  Because of your presence in the root OWNERS file you have been able to PR and merge code against this repo ever since it was created.
